CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+This Code of Conduct applies to Code.gov, its repositories, communications, events and anyone participating in the Code.gov community [link to about page]. Your use of Code.gov means you accept and agree to abide by this Code of Conduct, which we may revise periodically. You should review this page from time to time. 
```

Comment:
What does [link to about page] refer to?  Where should that link to?

Reply:
Not sure. @froi?